src/test/regress/sql/multi_modifying_xacts.sql

```diff
@@ -1157,4 +1157,25 @@ SELECT * FROM users JOIN usergroups ON (user_group = gid) WHERE id = 2;
 SELECT * FROM users JOIN usergroups ON (user_group = gid) WHERE id = 4;
 END;
 
+-- make sure functions that throw an error roll back propertly
```

Comment:
What do you think about adding tests for the following cases?
1. Call to another plpgsql function within a function.
2. Exception handling within a plplgsql function.